mingw-w64-rust/PKGBUILD

```
@@ -120,10 +136,22 @@ prepare() {
   # backport required for CLANG32 to build with compiler-rt 18.1.0
   (cd vendor/compiler_builtins && \
    apply_patch_with_msg \
-     67c1c0a71a204c089ddae4aec21ec75aa778c11b.patch)
+     d8ab794ed61e2c7c0750f57332a680d5aa8db48c.patch \
+     3f47913bc6414bab4254d49f9f6e7238fecace69.patch)
   # still needs to patch .cargo-checksums.json
   apply_patch_with_msg \
     0013-backport-compiler-builtins.patch
+
+  # patch tests
+  # tidy complains about *.orig files so do not generate backups
+  apply_patch_with_msg_no_backup \
+    0014-fix-vendored-test.patch \
+    8eb48b4f4c6e3d48f2600159a75184ec4d74b249.patch
+
+  if [[ $MINGW_PACKAGE_PREFIX == mingw-w64-x86_64 ]]; then
```

```suggestion
  if [[ $MSYSTEM == MINGW* ]]; then
```
Since this also seems to be necessary on MINGW32